src/Symfony/Component/Ldap/LdapClientInterface.php

```diff
@@ -18,12 +18,11 @@
  *
  * @author Grégoire Pineau <lyrixx@lyrixx.info>
  * @author Charles Sarrazin <charles@sarraz.in>
+ *
+ * @internal
  */
 interface LdapClientInterface
 {
-    const LDAP_ESCAPE_FILTER = 0x01;
-    const LDAP_ESCAPE_DN = 0x02;
```

Comment:
@nicolas-grekas Moreover, we want to prevent people from being highly coupled to an interface which will introduce BC at a later time. We may reintroduce them at a later date.